builtin/providers/aws/resource_aws_elb.go

```diff
@@ -271,6 +283,9 @@ func resourceAwsElbRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("listener", flattenListeners(lb.ListenerDescriptions))
 	d.Set("security_groups", lb.SecurityGroups)
 	d.Set("subnets", lb.Subnets)
+	d.Set("idle_timeout", lbAttrs.ConnectionSettings.IdleTimeout)
+	d.Set("connection_draining", lbAttrs.ConnectionDraining.Enabled)
+	d.Set("connection_draining_timeout", lbAttrs.ConnectionDraining.Timeout)
```

Comment:
Its not required because AWS defaults to 300. So the response in the read should always have a value.